testsuite/MDAnalysisTests/__init__.py

```diff
@@ -150,28 +147,4 @@
     in_dir,
     assert_nowarns,
 )
-from MDAnalysisTests.core.util import make_Universe
-
-def run(*args, **kwargs):
```

Comment:
modify this function so that I can still to `MDAnalysisTests.run()` in an interpreter.  This should start the pytest tests. I don't really care how the function is doing that. But it has to be rewritten.